mirage/seed_image/catalog_seed_image.py

```diff
@@ -168,9 +180,17 @@ def make_seed(self):
         if ((self.params['Inst']['mode'] in ['wfss','ts_wfss']) & \
             ('FULL' not in self.params['Readout']['array_name'])):
             self.seedimage, self.seed_segmap = self.pad_wfss_subarray(self.seedimage, self.seed_segmap)
-
+         
         # Save the combined static + moving targets ramp
         self.saveSeedImage()
+       
```

Comment:
Similarly, I think it would be cleaner if these lines were placed in a new function that is then called at the end of the main function.

Reply:
Bryan:  For this second change do you mean put the few lines of code that extract the central 2048x2048 pixels for the seed image and segmentation image into a new subroutine?